src/styles/main.scss

```diff
@@ -1,3 +1,2 @@
-body {
-  margin: 0;
-}
+@import "./initials.scss";
+@import "./calendar.scss";
```

Comment:
All is good in html. 
But you don't push import files on git. Make a `git add <file-path>` in your console, and push them to review,